test/e2e/specs/fakerp/etcdrecovery.go

```diff
@@ -0,0 +1,183 @@
+package fakerp
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+	batchv1 "k8s.io/api/batch/v1"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+	thefakeslimshady "github.com/openshift/openshift-azure/pkg/fakerp"
+	"github.com/openshift/openshift-azure/pkg/plugin"
+	"github.com/openshift/openshift-azure/pkg/util/log"
+	"github.com/openshift/openshift-azure/pkg/util/managedcluster"
+	"github.com/openshift/openshift-azure/pkg/util/randomstring"
+	"github.com/openshift/openshift-azure/pkg/util/ready"
+	"github.com/openshift/openshift-azure/test/clients/openshift"
+)
+
+var _ = Describe("Etcd Recovery E2E tests [EtcdRecovery][Fake][LongRunning]", func() {
+	const (
+		configMapName = "recovery-test-data"
+	)
+	var (
+		cli       *openshift.Client
+		admincli  *openshift.Client
+		backup    string
+		namespace string
+	)
+
+	BeforeEach(func() {
+		var err error
+		admincli, err = openshift.NewCustomerAdminClient()
```

Comment:
Not sure if customers should be able to restore clusters, this is something only SREs should be able to do?

Reply:
At least initially

Reply:
yes, it can't i get a permission error. I wasn't sure what user to use. The e2e passed, but it can't have run - not sure why

Reply:
Are we say this PR is flake because this should failled? and it didn't? 
/hold

Reply:
2018/12/04 10:18:35 Pod e2e-azure succeeded after 18m46s
the recover does quite a lot, i don't see how it could be done in 18m
does it list what tests it ran somewhere, I can't find this in the artifacts

Reply:
ok, i have not included the tag in the make e2e run, i'll add soon